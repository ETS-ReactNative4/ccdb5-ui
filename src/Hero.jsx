import './Hero.less';
import React from 'react';

export default class Hero extends React.Component {
  render() {
    return (
        <header className="content_hero">
          <h1>Consumer Complaint Database</h1>
          <p>
          Consumer complaints get added to this public database after the
          company has responded to the complaint (or after they've had the
          complaint for 15 calendar days, whichever comes first). We only
          publish the description of what happened if the consumer agrees to
          share it and after we take steps to <a>
            remove personal information
          </a>.
          Data is generally updated nightly.
          </p>
          <ul className="m-list m-list__horizontal">
            <li className="m-list_item"><a>Database Disclaimers</a> |</li>
            <li className="m-list_item"><a>How we use complaint data</a> |</li>
            <li className="m-list_item"><a>Technical Document</a></li>
          </ul>
        </header>
    );
  }
}
