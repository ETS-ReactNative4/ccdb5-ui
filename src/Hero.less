@import (less) "base.less";

.content_hero {
    padding: @gutter-normal;

    p {
        max-width: 670px;
    }

    ul {
        padding-top: @gutter-normal;

    }

    li {
        padding-bottom: 0;
        border-bottom: 0;

        a {
            margin-left: @gutter-narrow;
            margin-right: @gutter-narrow;
        }
    }
}