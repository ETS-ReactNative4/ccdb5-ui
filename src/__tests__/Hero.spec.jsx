import React from 'react';
import Hero from '../Hero';
import renderer from 'react-test-renderer';

describe('initial state', () => {
  it('renders without crashing', () => {
    const target = renderer.create(
      <Hero />
    );

    let tree = target.toJSON();
    expect(tree).toMatchSnapshot();
  });
});

