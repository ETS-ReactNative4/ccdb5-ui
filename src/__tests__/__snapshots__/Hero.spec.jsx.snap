exports[`initial state renders without crashing 1`] = `
<header
  className="content_hero">
  <h1>
    Consumer Complaint Database
  </h1>
  <p>
    Consumer complaints are added to this public database after the company has responded to the complaint, confirming a commercial relationship with the consumer, or after they\'ve had the complaint for 15 calendar days, whichever comes first. We donâ€™t verify all the facts alleged in complaints, but we do give companies the opportunity to publicly respond to complaints by selecting responses from a pre-populated list. Company-level information should be considered in the context of company size and/or market share.
  </p>
  <ul
    className="m-list m-list__horizontal">
    <li
      className="m-list_item">
      <button
        className="a-btn a-btn__link"
        onClick={[Function]}>
        More about the complaint database
      </button>
    </li>
    <li
      className="m-list_item">
      <a
        href="https://www.consumerfinance.gov/complaint/data-use/"
        target="_blank">
        How we use complaint data
      </a>
    </li>
    <li
      className="m-list_item">
      <a
        href="https://cfpb.github.io/api/ccdb/"
        target="_blank">
        Technical Documentation
      </a>
    </li>
    <li
      className="m-list_item">
      <a
        href="https://data.consumerfinance.gov/dataset/Consumer-Complaints/s6ew-h6mp"
        target="_blank">
        View complaint data in Socrata
      </a>
    </li>
  </ul>
</header>
`;
